packages/DropdownMenu/src/ContentContainer.styles.js

```diff
@@ -0,0 +1,19 @@
+import { css } from "styled-components";
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+
+const contentContainerStyles = css`
+  background: ${tokens.color.white};
+  border: 1px solid ${tokens.border.color};
+  border-radius: ${tokens.popover.borderRadius};
+  box-shadow: ${tokens.popover.shadow};
+  margin-left: ${stylers.spacer(2)};
+  max-width: ${tokens.popover.maxWidth}px;
+  min-width: 150px;
+  padding: ${tokens.space} 0;
+  text-align: left;
+
+  ${({ isOpen }) => (isOpen === false ? "opacity: 0;" : "")}
+`;
+
+export default contentContainerStyles;
```

Comment:
I think this file should be `DropdownMenu.styles.js`.  

And this export could easily be just `contentStyles`... but that's nitpicking.

Reply:
I have made it "DropdownMenuContentStyles.styles.js" as I think it is specific to the content of the dropdownMenu